lib/backend/syncersv1/bgpsyncer/bgpsyncer.go

```diff
@@ -57,9 +57,16 @@ func New(client api.Client, callbacks api.SyncerCallbacks, node string, watchAll
 			ListInterface:   model.ResourceListOptions{Kind: apiv3.KindBGPPeer},
 			UpdateProcessor: updateprocessors.NewBGPPeerUpdateProcessor(),
 		},
-		{
+	}
+	// When this syncer is used (via confd) in calico/node, it needs to know the affinity blocks
```

Comment:
the affinity blocks are actually just used to set up the blackhole routes - nothing to do with IPIP.

Reply:
addressed